monkey/monkey_island/deb-package/DEBIAN/control

```diff
@@ -5,4 +5,4 @@ Homepage: http://www.guardicore.com
 Priority: optional
 Version: 1.0
 Description: Guardicore Infection Monkey Island installation package
-Depends: openssl, python-pip, python-dev
+Depends: openssl, python3-pip, python3-dev
```

Comment:
does this guarantee 3.7?

Reply:
No. It guarantees pip compatible with py3 and py3 headers for pypi package compilation.